ruby/lib/minitest/queue/runner.rb

```diff
@@ -489,6 +489,10 @@ def parser
           opts.on('--failing-test TEST_IDENTIFIER') do |identifier|
             queue_config.failing_test = identifier
           end
+
+          opts.on('--execution_type type') do |type|
```

Comment:
I don't think we plan to have tons of these, so just `--fork` or `--enable-fork` would do I think.